src/index.js

```diff
@@ -2,6 +2,9 @@ import { create } from './create'
 import { run } from './run'
 
 const createWrapper = () => {
+  if ('undefined' === typeof window) {
+    return null;
```

Comment:
Also, please add some return to the user, if not an `console.error` at least a `console.log`